### PR TITLE
Fix all linter issues

### PR DESCRIPTION
**Pull request by sladyn98**
This PR fixes all linter issues per this comment. https://github.com/unifyai/ivy/pull/17864#issuecomment-1614930681

### CODE REVIEW

**Comment by sladyn98** [comment]
CC @KareemMAX I opened this PR lets see if it fixes stuff

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by sladyn98** [comment]
@KareemMAX It seems some of the lints are still failing .

**Comment by KareemMAX** [comment]
@sladyn98 Yes, can you please fix those?

File: ivy/functional/ivy/meta.py [hunk 1]
```diff
@@ -617,8 +620,10 @@ def reptile_step(
     ...     "latent": _variable(ivy.array([1.0, 2.0]))
     ... })
 
-    >>> cost, gradients, firsts = ivy.reptile_step(batch, inner_cost_fn, variables, 4, 0.025,
-    ...                                            batched=False, num_tasks=2, return_inner_v='first')
+    >>> cost, gradients, firsts = ivy.reptile_step(batch,
+                                                   inner_cost_fn, variables, 4, 0.025,
+    ...                                            batched=False,
+                                                   num_tasks=2, return_inner_v='first')
```
Comment (KareemMAX) [inline ivy/functional/ivy/meta.py:626]: Note that this is an invalid documentation. Python examples should start with `>>>` or `...`

**Review by KareemMAX — CHANGES_REQUESTED** [review]

File: ivy/functional/ivy/meta.py (hunk 1 above)
Comment (sladyn98) [reply ivy/functional/ivy/meta.py:626]: thanks

**Comment by sladyn98** [comment]
docformatter.............................................................Failed

Not sure why docformatter is still failing

**Comment by KareemMAX** [comment]
@sladyn98 `doc-formatter` should update files to fail.

**Comment by sladyn98** [comment]
@KareemMAX Do i need to run this ?

**Comment by KareemMAX** [comment]
@sladyn98 Yup you should run it.

Also I guess I'm a bit slow on replying. I guess we can move that to a private discord conversation if you like

**Comment by sladyn98** [comment]
@KareemMAX Messaged you on discord, seems like running this command does not work

**Comment by KareemMAX** [comment]
I'll close this PR for now as I guess your branch is very old and some of the lint errors you fixed appear as new code that you are attempting to add [like here](https://github.com/unifyai/ivy/pull/18124/files#diff-c54ef962e94798f67898c1def7eadded4b4b32f3822502df9e4edaec293c3204R58-R61). I'll fix lint errors in master directly

**Pull request closed without merging**